hosts/prbuilder/configuration.nix

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+#
+# SPDX-License-Identifier: Apache-2.0
+{
+  self,
+  inputs,
+  lib,
+  pkgs,
+  ...
+}: {
+  imports = lib.flatten [
+    (with inputs; [
+      nix-serve-ng.nixosModules.default
+      disko.nixosModules.disko
+    ])
+    (with self.nixosModules; [
+      common
+      qemu-common
+      service-openssh
+      user-jrautiola
+      user-cazfi
+      user-karim
+    ])
+    ./disk-config.nix
+  ];
+  nixpkgs.hostPlatform = lib.mkDefault "x86_64-linux";
+   # List packages installed in system profile
+  environment.systemPackages = with pkgs; [
+    git
+    docker
```

Comment:
docker is already installed by the `virtualisation.docker.enable` option so this is not needed.